app/controllers/application_controller.rb

```diff
@@ -184,8 +206,8 @@ def setup_request_context
 
     Rails.logger.info "  Host #{host_name}, community ##{RequestContext.community_id} " \
                       "(#{RequestContext.community&.name})"
-    unless RequestContext.community.present?
-      render status: 422, plain: "No community record matching Host='#{host_name}'"
+    if RequestContext.community.blank?
```

Comment:
Question (curiosity): Why this change?

Reply:
Rubocop... don't ask me _why_ it prefers this way :P